utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -97,7 +98,8 @@ std::map<int, string> augmentationMap =
     {84, "spatter"},
     {85, "swap_channels"},
     {86, "color_to_greyscale"},
-    {87, "tensor_sum"}
+    {87, "tensor_sum"},
+    {92, "bitwise_or"}
```

Comment:
Why is the case number for bitwiseOR this far apart from bitwiseAND? Pls check BatchPD case numbers.

Reply:
Modified testCase of Bitwise OR to 68 to match with Inclusive OR of BatchPD.